src/patterns/atoms/forms/collection.yaml

```diff
@@ -0,0 +1,4 @@
+title: Forms
+labels:
+  - inprogress
+notes: Basic form elements.
```

Comment:
Is this used anywhere? (I'm still trykng to understand how drizzle all works under-the-hood)

Reply:
It's not currently displayed anywhere but I filed https://github.com/mozilla/protocol/issues/1 ages ago. Eventually it would be good to have a description/introduction for the entire collection but right now it's useless here. I can remove it.